tests/test_operator.py

```diff
@@ -8,9 +8,11 @@
 from kubernetes.client.rest import ApiException
 
 TEST_DATA = [{"type": "CREATED", "object": {"metadata": {"name": "hub"},
-                                      "spec": {"upstreamUrl": "hubtest"}}},
+                                      "spec": {"upstreamUrl": "hubtest",
+                                                "credentialsSecret": "internal-mirror"}}},
              {"type": "MODIFIED", "object": {"metadata": {"name": "hub"},
-                                      "spec": {"upstreamUrl": "hubtest"}}}]
+                                      "spec": {"upstreamUrl": "hubtest",
+                                               "credentialsSecret": "internal-mirror"}}}]
```

Comment:
the previous test didn't actually check what happened when you had a credentialsSecret, which lowered the code coverage.